src/github.com/getlantern/flashlight/config/bootstrap.go

```diff
@@ -102,14 +103,15 @@ func MakeInitialConfig(configPath string) error {
 	bytes, err := fs.GetIgnoreLocalEmpty("lantern.yaml")
```

Comment:
This is using tarfs, which can include an embedded version.  This is existing code which I didn't change, so I'm assuming it does what it's supposed to.